pkg/install/install.go

```diff
@@ -124,33 +138,37 @@ func NewInstaller(ctx context.Context, log *logrus.Entry, env env.Interface, db
 
 // Install installs an ARO cluster
 func (i *Installer) Install(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
-	steps := map[api.InstallPhase][]func(context.Context) error{
+	steps := map[api.InstallPhase][]interface{}{
 		api.InstallPhaseDeployStorage: {
-			i.createDNS,
-			func(ctx context.Context) error {
+			action(i.createDNS),
+			action(func(ctx context.Context) error {
```

Comment:
I'm wondering if I should define this function somewhere, as currently the logging output is:
```
running step github.com/Azure/ARO-RP/pkg/install.(*Installer).Install.func1
```